src/components/ModalDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 -2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+import type { ReactNode } from 'react'
+
+import { ModalContent } from './components/ModalContent'
+import { ModalHeader } from './components/ModalHeader'
+import { StyledFormFooter, StyledModal } from './styled'
+
+export type ModalDialogProps = {
+  onClose: () => void
+  title: string
+  onConfirm?: () => void
+  children?: ReactNode
+  isConfirmation?: boolean
+  open?: boolean
+}
+
+const ModalDialog = ({
+  open = true,
+  onClose,
+  onConfirm,
+  title,
+  isConfirmation = false,
+  children,
+}: ModalDialogProps) => (
+  <StyledModal open={open} onClose={onClose} data-testid="modal-dialog">
+    <ModalHeader title={title} onClose={onClose} />
+    <ModalContent>{children}</ModalContent>
+    {isConfirmation && (
+      <StyledFormFooter
```

Comment:
This is one of the typical use cases for a padding. The padding should be on the buttons. Currently there are two margins defined which causes the footer component to not touch the bottom of the container. Also currently when making the window smaller, there is no space between the text and the button. A padding should fix this.
<img width="659" alt="Screenshot 2023-08-09 at 10 44 32" src="https://github.com/Amsterdam/signals-frontend/assets/7532781/2957ee85-d3bb-4ed2-91fd-1e06211997b3">

Reply:
<img width="672" alt="Screenshot 2023-08-09 at 10 48 59" src="https://github.com/Amsterdam/signals-frontend/assets/7532781/fd0bdc0c-ec93-465f-84b6-be584dca66a8">

Reply:
Right, the difference between padding and margin keeps eluding me. Here I thought margin would be applicable because it's not in relation to another another component. I've changed it to padding.